policies/trademark-policy.md

```diff
@@ -1,172 +1,296 @@
-Any use of or reference to the Commonhaus Foundation trademarks that is inconsistent with this trademark policy, or use of marks that are confusingly similar to trademarks of the Commonhaus Foundation, is prohibited.
+#### Distribution of unmodified source code or artifacts packaged by us
 
-All uses of Commonhaus Foundation trademarks, and all goodwill associated therewith, will inure solely to the benefit of the Commonhaus Foundation.
+When you redistribute an unmodified copy of our software, either in source code form or as packaged artifacts (such as executables, libraries, or jar files), you are not changing the quality or nature of the software.
 
-### Acknowledgment and Symbols
+In this case, you may retain the Word Marks and the Logos we have placed on the software to identify your redistribution.
+This kind of use only applies if you are redistributing an official distribution from this Project that has not been changed in any way.
 
-- Use the CF Marks in a form that distinguishes them from the text around them, such as by capitalization, bold or italic fonts, or with quotation marks.
+You can find files for the Logos and approved designs for packaging on our [Brand Standards][] page.
 
-- Use the appropriate symbol (® for registered trademarks; ™ for unregistered; ℠ for unregistered service marks) next to the first use of CF Marks.
-  For a detailed list, visit [Commonhaus Trademarks](https://www.commonhaus.org/trademarks/).
+> [!NOTE]
+>
+> When we refer to "our software," we mean the software produced by CF Projects.
+> "Artifacts packaged by us" refers to software artifacts packaged and published by each CF Project, using their own processes and infrastructure.
 
-- Use the full form of any trademarks (e.g., Commonhaus Foundation”) in the first reference in all documents of mass communication, including marketing collateral and web pages.
-  You may then use any abbreviated or short form references (e.g. CF) within the same document where the full form has already been used.
+#### Distribution of modified source code or artifacts packaged you
 
-### Compatibility and Interoperability
+If you compile and distribute bitwise identical packaged artifacts from our unmodified source code, you may retain our Word Marks and Logos.
+
+You may use the Word Marks, but not the Logos, to truthfully describe the origin of the software that you are providing, that is, that the code you are distributing is a modification of our software. You may also say, for example, that "this software is derived from the source code for Hibernate version 5.2."
+
+If your modification includes minor changes like bug fixes that preserve compatibility with our published artifacts, your artifacts should be labeled to indicate they were built by you.
+For instance, the Maven dependency io.quarkus:quarkus-arc:3.8.5.redhat-00004 clearly indicates it was built by Red Hat.
```

Comment:
```suggestion
For instance, the Maven dependency io.quarkus:quarkus-arc:3.8.5.redhat-00004 clearly indicates it was built by Red Hat.
```

I get where this is coming from but I don't think it can be put in exactly like this; because it depends on the packaging system and mechanism applied if that name matters.

i.e. in go-lang symbolic names are used and one way for a company red hat or otherwise would be forced to provide a supported build would be to use the same physical name to distribute under but from a different repository.  Thus the source you get it from will have a different name (i.e. maven.repository.redhat.com).

Reply:
removed the example. Left with generic "labeled"